package com.leaveloper.habitsappcourse.ui.theme

import android.app.Activity
import android.os.Build
import androidx.compose.foundation.isSystemInDarkTheme
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.darkColorScheme
import androidx.compose.material3.dynamicDarkColorScheme
import androidx.compose.material3.dynamicLightColorScheme
import androidx.compose.material3.lightColorScheme
import androidx.compose.runtime.Composable
import androidx.compose.runtime.SideEffect
import androidx.compose.ui.graphics.toArgb
import androidx.compose.ui.platform.LocalContext
import androidx.compose.ui.platform.LocalView
import androidx.core.view.ViewCompat

private val ColorScheme = lightColorScheme(
    primary = Primary,
    secondary = Background,
    tertiary = Accent,
    background = Background,
    onPrimary = Accent,
    onSecondary = Primary,
    onBackground = Primary,
    onTertiary = Primary,
    surface = Background,
    onSurface = Accent
)

@Composable
fun HabitsAppCourseTheme(
    content: @Composable () -> Unit
) {
    MaterialTheme(
        colorScheme = ColorScheme,
        typography = Typography,
        content = content
    )
}